independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/Methods.java

```diff
@@ -105,7 +106,13 @@ static void addDelegatingMethods(IndexView index, ClassInfo classInfo, Map<TypeV
     }
 
     private static boolean skipForClientProxy(MethodInfo method) {
-        if (Modifier.isStatic(method.flags()) || Modifier.isFinal(method.flags()) || Modifier.isPrivate(method.flags())) {
+        short flags = method.flags();
+        String className = method.declaringClass().name().toString();
+        if (Modifier.isFinal(flags) && !className.startsWith("java.")) {
```

Comment:
Hm, is there actually a reason to skip the message for JDK classes?

Reply:
To keep the logs to a minimum user related stuffs.

Reply:
Ok makes sense. User can define a producer for a JDK class but it's probably reasonable.